crates/burn-candle/src/lib.rs

```diff
@@ -86,6 +86,7 @@ mod tests {
     burn_tensor::testgen_mul!();
     burn_tensor::testgen_neg!();
     burn_tensor::testgen_permute!();
+    burn_tensor::testgen_remainder!();
```

Comment:
Comment this out for now and add a note explaining the observed failure on macos.